File: C++/A-ch1-4-奇偶數加總.cpp
#include <iostream>
using namespace std;

int main() {
    int i, n, sum_odd=0, sum_even=0;
    cout << "請輸入數字:";
    cin >> n;
    for (i=1; i<=n; i++){
        if (i % 2 == 1) 
            sum_odd = sum_odd + i;
        else
            sum_even= sum_even +i;
    }
    cout << "奇數總和:" << sum_odd  << endl;
    cout << "偶數總和:" << sum_even << endl;
    
    return 0;
}

File: Python/a_ch1_4_奇偶數加總.py
sum_odd=0
sum_even=0
n = int(input("請輸入數字:"))

for i in range(1, n+1):
  if i % 2 == 1:
    sum_odd = sum_odd + i
  else:
    sum_even = sum_even + i

print(sum_odd)
print(sum_even)